### DISCUSSION TITLE
Parece que mudaste outra vez os links da TVI, não querias os originais?

### DISCUSSION THREAD

**Discussion by thomraider12** [inspirationlinks/lista-tuga#13] [General]
Ele alterava te os links sozinho.

**Comment by iprtl** [comment]
nao é isso. Passava o tempo a receber msg de erro. Alem disso os links como estao na lista, nao me parece que atualizassem automaticamente.

EXEMPLO:
`#EXTINF:-1 tvg-name="" tvg-id="tvi.internacional.pt" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/tviInternacional.webp",TVI Internacional
https://raw.githubusercontent.com/ipstreet312/freeiptv/master/ressources/tvipt/sh/tviint.m3u8

#EXTINF:-1 tvg-name="" tvg-id="TVI.HD.pt" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/tvi.webp",TVI
https://raw.githubusercontent.com/ipstreet312/freeiptv/master/ressources/tvipt/sh/tvi.m3u8

#EXTINF:-1 tvg-name="CNN Portugal" tvg-id="CNN.Portugal.HD.pt" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/CNNPortugal.webp",CNN Portugal 
https://raw.githubusercontent.com/ipstreet312/freeiptv/master/ressources/tvipt/sh/cnnpt.m3u8

#EXTINF:-1 tvg-name="" tvg-id="V+TVI.pt" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/V%2Btvi.webp",V+ TVI
https://video-auth2.iol.pt/live_vmais/live_vmais/edge_servers/vmais-720p/playlist.m3u8
#EXTINF:-1 tvg-name="" tvg-id="V+TVI.pt" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/V%2Btvi.webp",V+ TVI 
https://bit.ly/3gejKxf?profile=inspirationFreetv.m3u8

#EXTINF:-1 tvg-name="" tvg-id="tvi.africa.pt" tvg-name="" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/tvi%C3%81frica.webp",TVI África
https://video-auth4.iol.pt/live_tvi_africa/live_tvi_africa/edge_servers/tviafrica-480p/playlist.m3u8

#EXTINF:-1 tvg-name="" tvg-id="TVI.Reality.HD.pt" group-title="Portugal 🇵🇹" tvg-logo="https://github.com/inspirationlinks/m3u/raw/live/LogosTv/tviReality.webp",TVI Reality
https://video-auth5.iol.pt/live_tvi_reality/live_tvi_reality/edge_servers/tvireality-720_passthrough/playlist.m3u8`

**Reply by thomraider12** [reply]
Não atualizaram automaticamente porque os mudaste para estes: 


https://raw.githubusercontent.com/ipstreet312/freeiptv/master/ressources/tvipt/sh/tviint.m3u8
https://raw.githubusercontent.com/ipstreet312/freeiptv/master/ressources/tvipt/sh/tvi.m3u8
https://raw.githubusercontent.com/ipstreet312/freeiptv/master/ressources/tvipt/sh/cnnpt.m3u8

**Comment by thomraider12** [comment]
Assim dá te sempre erro

**Comment by iprtl** [comment]
Entao qual os links exatamente que tenho que colocar?
Estes??
https://video-auth6.iol.pt/live_tvi/live_tvi/playlist.m3u8
https://video-auth7.iol.pt/live_cnn/live_cnn/playlist.m3u8
https://video-auth6.iol.pt/live_tvi_internacional/live_tvi_internacional/playlist.m3u8

**Comment by thomraider12** [comment]
Sim. Depois de os colocares, não tens de fazer mais nd.

**Comment by iprtl** [comment]
ok podes manda r o script outra vez??

**Reply by thomraider12** [reply]
Ja não precisas, porque o script já está a funcionar. É automático. Agora, não mudas nada nesses canais.
